src/otx/recipe/_base_/data/mmaction_base.yaml

```
@@ -6,6 +6,7 @@ config:
     - 500
     - 500
   image_color_channel: RGB
+  unannotated_bg_ratio: 0.0
```

```suggestion
  unannotated_items_ratio: 0.0
```
I'm feeling that "unannotated" and "bg" are semantically duplicated each other.

---

Thank you for the suggestion. - https://github.com/openvinotoolkit/training_extensions/pull/2965/commits/ba350f2b7cbedf68e7306874b7e02a7ec36a95e4